ui/src/pages/Configuration/ConfigurationPage.jsx

```diff
@@ -103,6 +104,7 @@ function ConfigurationPage() {
                         <ColumnLayout.Column span={9}>
                             <TitleComponent>{_(title)}</TitleComponent>
                             <SubTitleComponent>{_(description || '')}</SubTitleComponent>
+                            <SubDescription {...subDescription} />
```

Comment:
is it null safe?

Reply:
yea, if null/undefiend then props are empty object and `<></>` is returned, as there is additional required check 
```
    if (!props?.text) {
        return <></>;
    }
```
which unfortunately is mandatory due to `text = ''`, as that would resolve in empty div with margin etc.